darts/tests/test_timeseries_generation.py

```diff
@@ -94,3 +94,22 @@ def test_routine(time_index, country_code):
             for country_code in ['US', 'CH', 'AR']:
                 test_routine(time_index, country_code)
 
+        # test extend time index
+        test_routine(time_index_1, 'US',  add_length= 365)
```

Comment:
```suggestion
        test_routine(time_index_1, 'US',  add_length=365)
```

Reply:
Ah also, any specific reason for why you put 2 whitespaces between function arguments?